xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_macros.vm

```diff
@@ -57,7 +57,6 @@
 ###
 #macro(xwikitopmenuleftstart)
   <ul class="nav navbar-nav navbar-left">
-    #submenuseparator()
```

Comment:
The question is debatable, I think it's okay, and it passes the axe-core tests we've set up.
There's a discussion about this subject, I find the answer a bit unclear but it's here: https://stackoverflow.com/questions/71754434/is-an-html-list-ul-or-ol-allowed-to-be-empty-for-accessibility
---
What I got out of it:
- Only the siteImprove tool reports (reported?) this as a violation
- The WCAG is not clear on this point
- Screenreaders will ignore empty lists (just like they are often ignored in presentation)

---
My opinion and choices so far: 
- I left it in because I didn't want to break the interface of some obscure extension that added an item to this list
- An empty list is semantically correct if labelled (e.g. list of the students in the classroom <empty> makes sense, so elements in the left navbar <empty> should be correct too) 
---
The solutions I see:
1. The navbar left list is useless, removing it from the DOM won't cause retro compatibility issues. We remove it.
2. The navbar left is not completely useless, we keep it and label it.

Reply:
> The navbar left is not completely useless, we keep it and label it.

I think that's the safest. But how do we label it?

Reply:
Since AFAIK we don't have any idea of the content it would contain, we can label it with a description:
e.g. "Left side of the navigation bar."
This way: 
- 0 item in the list can be understood as "there is nothing in the left side of the navigation bar" (similarly to the example above with students in a classroom).
- The list itself, if it's read/displayed somehow, is a "List: left side of the navigation bar"
This is a bit technical for a title but I don't know how we'd do better.